src/winshellext/F3DThumbnailProvider.cxx

```diff
@@ -93,6 +94,14 @@ HRESULT CreateBitmapFromBuffer(vtkImageData* image, HBITMAP* phbmp)
 
   return NOERROR;
 }
+
+//------------------------------------------------------------------------------
+void SignalHandler(int signum)
+{
+  std::cerr << "Interrupt signal (" << signum << ") received.\n";
```

Comment:
[Doc](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/signal?view=msvc-170) says
"Do not issue low-level or STDIO.H I/O routines (for example, printf or fread)." so you should remove this (useless anyway) output.

Reply:
good point